src/cli/cmds/synth.ts

```diff
@@ -94,6 +94,7 @@ class Command implements yargs.CommandModule {
         const app = await synthApp(command, tempDir, stdout, recordConstructMetadata);
         for (const f of app.manifests) {
           fs.createReadStream(f).pipe(process.stdout);
+          process.stdout.write('---\n');
```

Comment:
This won't work as expected because the line just above it as asynchronous. It will print this before the manifests are printed. To do this we first need to await on the created pipeline:

```typescript
import { pipeline } from 'stream';
import { promisify } from 'util';

const asyncPipeline = promisify(pipeline);

await asyncPipeline(fs.createReadStream(f), process.stdout, { end: false });
process.stdout.write('---\n');
```

We should also have test to assert on the content of `process.stdout` using jest spies or something.